client/charts-directive.js

```diff
@@ -50,21 +50,7 @@ var renderStringChart = function(scope, dimensions) {
      .attr("y", function(d, i) { return yScale(i); })
      .attr("width", function(d) { return xScale(d.freq); })
      .attr("height", yScale.rangeBand())
-     .on("mouseover", function(d){
-       var tooltip = d3.select("body")
```

Comment:
For some reason at the moment I have a change pending to this line in my local version that changes it to:

`var tooltip = d3.select(divID)`

I don't know why I made this change. :( It's been a while. 

Just saying ... for posterity.